### PR TITLE
Update configuring.tex

### PR DESCRIPTION
**Pull request by MauriceMEGA65**
Added explanation on boot sequence of essential files.
ps, I'm trying to upload the picture

### CODE REVIEW

**Comment by MauriceMEGA65** [comment]
Please rename picture text (all the way down in my proposal) to 'mega65_OpenROM_boot_noSD.png'

**Comment by gardners** [comment]
Hello,
You will need to make the changes yourself, as I can't edit a pull request owned by you. See this article for instructions:

https://zellwk.com/blog/edit-pull-request/

I'll add some comments to the changes as well.

Also, I think you still need to `git add` and `git commit` the PNG file you added, as it hasn't showed up in the pull request.

File: configuring.tex [hunk 1]
```diff
@@ -97,6 +97,25 @@ \section{Preparing for First Use: Formatting SD Cards}
 Record (``Boot Sector'') by instead typing ``FIX MBR,'' should the
 need arise.
 
+
+As an end result, we want to have a correctly formatted SD card with the essential files stored on it for MEGA65 to boot from.
+
+This is how it works: When powering on, MEGA65 will search for- and boot these files:
+o FREEZER.M65
```
Comment (gardners) [inline configuring.tex:104]: Lists need to look like this:

\begin{itemize}
\item {\tt FREEZER.M65}
\item {\tt AUDIOMIX.M65}
...
\end{itemize}

The {\tt ...} wrapper selects the C64 screen font which we use when mentioning filenames.

File: configuring.tex [hunk 2]
```diff
@@ -97,6 +97,25 @@ \section{Preparing for First Use: Formatting SD Cards}
...
+o C65THUMB.M65
+o MEGA65.ROM
+o MEGA65.D81 (default disk image, automatically mounted during start)
+
+Straight out-of-the-box, MEGA65 will only have one SD card installed, located inside behind the trap-door. This SD card contains the essential files to properly boot from.
+When an external microSD card is inserted, MEGA65 consider the external one as higher priority and will try to boot from it.
+That means that the microSD card needs to have the essential files on it, otherwise MEGA65 cannot boot properly and will fall back into loading the OpenROM which does not support all the MEGA65 features.
+In general, if MEGA65 cannot boot properly and fall back to OpenROM, it will be shown in the power-up screen, similar to this:
+
+\includegraphics[width=\linewidth]{images/mega65-OpenROM boot-noSD.png}
```
Comment (gardners) [inline configuring.tex:116]: As we discussed, you wil need to make sure there are no spaces in the file name. The -'s are ok, though.

**Review by gardners — CHANGES_REQUESTED** [review]
Hello again Maurice,

Thanks for making your first pull request.  Don't be discouraged by the process or changes. Once you have done it a couple of times it will be super easy, and we really value your contributions!

Paul.

File: configuring.tex (hunk 1 above)
Comment (MauriceMEGA65) [reply configuring.tex]: Done

File: configuring.tex (hunk 2 above)
Comment (MauriceMEGA65) [reply configuring.tex]: Done

**Review by MauriceMEGA65 — COMMENTED** [review]
Updated with comments Paul

File: configuring.tex [hunk 3]
```diff
@@ -97,6 +97,27 @@ \section{Preparing for First Use: Formatting SD Cards}
...
+As an end result, we want to have a correctly formatted SD card with the essential files stored on it for MEGA65 to boot from.
+
+This is how it works: When powering on, MEGA65 will search for- and boot these files:
+\begin{itemize}
+\item {\tt FREEZER.M65}
+\item {\tt AUDIOMIX.M65}
+\item {\tt C64THUMB.M65
+\item {\tt C65THUMB.M65
+\item {\tt MEGA65.ROM
+\item {\tt MEGA65.D81 (default disk image, automatically mounted during start)
```
Comment (gardners) [inline configuring.tex:110]: Close the {\tt ...} after the file name, so that the comment after will be rendered in the normal font.

**Review by gardners — CHANGES_REQUESTED** [review]
Hello,
Mostly there. Just one further comment, and I still can't see the PNG file in the pull request. Did you try `git add` and `git commit` on that new file?
Paul.

**Review by gardners — APPROVED** [review]
ok, looks good now. Well done for persevering!

**Pull request merged**